src/manager/agent.config.ts

```diff
@@ -133,6 +137,7 @@ export class AgentConfig {
             '--webhook-url', this.webhookUrl,
             '--log-level', this.logLevel,
             '--wallet-local-did', // TODO this could be an arg on the config
+            '--tails-server-base-url', this.tailsUrl,
```

Comment:
Have you tested how aca py reacts to a null tails url? My guess is it won't like it. You may need to add an if statement like the if (this.seed) logic below.